nginx.conf

```diff
@@ -30,6 +30,11 @@ server {
       add_header Cache-Control "no-store, no-cache, must-revalidate";
     }
 
+   location /assets {
+      try_files $uri $uri/ =404;
+      add_header Cache-Control "public, max-age=3600";
```

Comment:
We could consider a higher cache duration, e.g. 1 day to 1 week, since these assets are likely to not change that frequently

Reply:
agreed 👍 @Planlos5000 and I were still looking into it since on the dev deployment, this location rule doesn't seem to work right now